### DISCUSSION TITLE
请教

### DISCUSSION THREAD

**Discussion by admintony** [howmanybots/onebot#138] [问答]
目前来讲，这只是一套标准，还是已经完成了这套SDK的编写了？如果已经完成，请给出代码地址，十分感谢，

**Comment by stdrc** [comment]
@admintony 目前是标准，需要各兼容项目来实现，相关项目在文档的“生态”页面有

**Comment by gitlsl** [comment]
有时候机器人传文件只能传本地的文件吧,     所以得有一套 跟机器人所在电脑进行 文件上传下载的接口?..

**Comment by stdrc** [comment]
> 有时候机器人传文件只能传本地的文件吧, 所以得有一套 跟机器人所在电脑进行 文件上传下载的接口?..

目前基本上所有 OneBot（CQHTTP）实现都是可以跑在 Linux 的，所以不存在以前那种机器人后端在 Linux、酷Q 必须跑在 Windows 的情况了，因此不太考虑加入文件传输接口

**Comment by gitlsl** [comment]
可以备选嘛,  毕竟这个需求还是挺正当的
另外bot其他接口方面 可以参考一些 telegram 的~

**Comment by ExerciseBook** [comment]
感觉也有人在考虑加入其他规则（Discord  啊 Telegram 啊 之类的）的支持？

**Comment by stdrc** [comment]
@ExerciseBook 我认为可以看发展情况，v12 可以再做一个改动，让字段更通用（比如去掉所有 qq 和 user_id 改成字符串）
